### ISSUE TITLE
Deploy a CAP Business Application to BTP: The plan hdi-shared could not be found for service hanatrial

### ISSUE THREAD

**Issue by PierreFritsch** [SAPDocuments/Issues#347]
Tutorials: https://developers.sap.com/tutorials/cap-service-deploy.html
--------------------------

In [Step 3: Deploy using cf push](https://developers.sap.com/tutorials/cap-service-deploy.html#2eb18b1c-472a-4489-9b73-03140696c53e)
when I run:

    cf create-service hanatrial hdi-shared my-bookshop-db

then I get the error:

> Creating service instance my-bookshop-db in org XXXXXXtrial / space dev as XXX@sap.com...
> The plan hdi-shared could not be found for service hanatrial.
> FAILED

I would suggest to update the Tutorial so that it contains the current service plan that should be used.

----

**Hint** 

When I run:

> cf marketplace | grep hana

then I get the following service plans:

* `hana` (`hdi-shared`, `sbss`, `schema`, `securestore`) _Manage schemas and HDI containers on an existing SAP HANA database._
* `hanatrial` (`securestore`) _Manage Schemas and HDI Containers on a shared SAP HANA database_ 
* `hana-cloud-trial` (`hana`, `relational-data-lake`, `hana-cloud-connection`) _Leverage the in-memory data processing capabilities of SAP HANA in the cloud as one simple gateway to all data._

**Comment by iwonahahn** [comment]
Hi @PierreFritsch,

Which version of `@sap/cds-dk` and `Node.js` are you using? Can you please provide the output of `cds -v`? 

Thanks a lot and best regards,
Iwona

**Comment by PierreFritsch** [comment]
Hi @iwonahahn 

Not sure how relevant this is, but here you go:

```
my-bookshop % node -v
v14.16.1
my-bookshop % cds -v
@sap/cds: 5.0.6
@sap/cds-compiler: 2.1.6
@sap/cds-dk: 4.0.7
@sap/cds-foss: 2.3.0
@sap/cds-runtime: 3.0.9
Node.js: v14.16.1
home: /Users/XXXXXX/Desktop/my-bookshop/node_modules/@sap/cds
my-bookshop: 1.0.0
``` 

Probably more interesting:

```
my-bookshop % cf api
API endpoint:   https://api.cf.eu10.hana.ondemand.com
API version:    3.97.0
```

Cheers
Pierre

**Comment by PierreFritsch** [comment]
... to connect the dots: 
For me, the following command does the trick:

     cf create-service hanatrial securestore my-bookshop-db

**Comment by MinionBee** [comment]
Hi @PierreFritsch.
Currently, I'm also getting the same error when I deploy the MTA project. 
In my BTP trial account, service **SAP HANA Schemas and HDI Containers trial** not contain plan **hdi-shared**.  
Did you resolve this error? If yes, could you please share the steps to resolve the issue?
Thanks.

**Comment by iwonahahn** [comment]
Hi @PierreFritsch,

Sorry for the late reply. I will test it with the latest CAP release and discuss it with the development colleagues and will get back to you latest next Monday (due public holiday in Germany). 

Best regards,
Iwona

**Comment by iwonahahn** [comment]
Hi @PierreFritsch,

the service SAP HANA Schemas & HDI Containers Trial (`hanatrial`) was removed for newly created trial accounts. You now need to use the trial service on SAP HANA Cloud (`hana-cloud-trial`) to deploy your application. To avoid confusion we will update the tutorial asap.

Best regards,
Iwona

**Comment by iwonahahn** [comment]
Hi @PierreFritsch,

the service SAP HANA Schemas & HDI Containers Trial (`hanatrial`) was removed for newly created trial accounts. You now need to use the trial service on SAP HANA Cloud (`hana-cloud-trial`) to deploy your application. To avoid confusion we will update the tutorial asap.

Best regards,
Iwona

**Comment by iwonahahn** [comment]
Hi @PierreFritsch,

the service SAP HANA Schemas & HDI Containers Trial (`hanatrial`) was removed for newly created trial accounts. You now need to use the trial service on SAP HANA Cloud (`hana-cloud-trial`) to deploy your application. To avoid confusion we will update the tutorial asap.

Best regards,
Iwona

**Comment by iwonahahn** [comment]
Hi @PierreFritsch,

I have tested it with a newly created trial subaccount and you are right. For the service SAP HANA Schemas & HDI Containers Trial the plan `securestore` can be used. It works with `cf create-service hanatrial securestore my-bookshop-db` in step 3.1. I have updated the tutorial today. Thank you for reporting and testing it :). 

@minionbee: In step 3.1 try 

`cf create-service hanatrial securestore my-bookshop-db`

instead of

`cf create-service hanatrial hdi-shared my-bookshop-db`

This should work.

Best regards,
Iwona
